test/conformance/adapter/adapter_adapter_native_cpu_loader.match

```diff
@@ -0,0 +1 @@
+adapter_adapter_native_cpu.match
```

Comment:
are these somehow referencing the existing match file? did we already have something in the scripts that let us do that?

Reply:
This is a symlink. Github doesn't display them very well except in the sidebar.

Reply:
aaah I see, I wonder if we could just stop doing the `EXTRA_NAME` thing here https://github.com/oneapi-src/unified-runtime/pull/2168/files#diff-1595fcdcdd0e6bf22312a2db5cbf98fb89265acbbc4bfa9887845dbe193a291fR22 I can't see any cts match files actually using it to have multiple different match files for a given adapter/test combo, and honestly I wouldn't want us to start doing that - the whole thing is fiddly enough as is

Reply:
EXTRA_NAME seems to be unused and so could probably be removed. But I think that should be a separate change.

Reply:
isn't EXTRA_NAME the thing that's necessitating all these symlinks? what I'm getting at is it would be cleaner if we could avoid the clutter by changing how the targets are set up

Reply:
EXTRA_NAME is unused, the symlinks are created because of the first parameter ("name"). I'm intentionally giving them a different match file because its possible that the loader could introduce (or fix) specific tests.

... Although, now I think about it, `{{opt}}` might be a better solution to that.

Reply:
ah right, yeah I'd prefer leaving that for `{{opt}}` if it turns out to be necessary